src/scss/skin-modern/components/_listbox.scss

```diff
@@ -0,0 +1,35 @@
+@import '../variables';
+@import '../mixins';
+
+%ui-listbox {
+  @extend %ui-container;
+
+  .#{$prefix}-ui-listboxitem-wrapper {
+    &.#{$prefix}-ui-listboxitem-wrapper-selected {
+      background-color: transparentize($color-highlight, .3);
+    }
+
+    .#{$prefix}-ui-listboxitem-button {
+      @extend %ui-button;
+
+      font-size: .8em;
+      height: 100%;
+      padding: .5em 2em;
+      width: 100%;
+
+      .#{$prefix}-label {
+        display: inherit;
+      }
+
+      &:hover {
+        background-color: transparentize($color-highlight, .15);
+      }
+
+      &:last-child {
+        border-bottom: 0;
+      }
+    }
+  }
+}
+
```

Comment:
You are missing the actual class declaration here:

```scss
.#{$prefix}-ui-listbox {
  @extend %ui-listbox;
}
```

Without that, a basic `ListBox` would not be styled.